Sketchbook/CRSF_Joystick/CRSF_Joystick.ino

```diff
@@ -4,16 +4,20 @@
 #include "calibration.h"
 
 // ONLY UNCOMMENT 1 Board ID
-//#define BOARD_ID 1  // PiPico
+#define BOARD_ID 1  // PiPico

```

Comment:
Need to comment out board id 1.  The define below will technically override it, but worth keeping it clean.